tests/test_importers.py

```diff
@@ -18,11 +19,12 @@ def setUp(self) -> None:
         pass
 
     def tearDown(self) -> None:
-        single_level_file = os.path.join(FILE_PATH, "single_level.db")
+        single_level_file = os.path.join(CURRENT_DIR, "single_level.db")
         if os.path.exists(single_level_file):
             os.remove(single_level_file)
 
-        descending_file = os.path.join(FILE_PATH, "descending.db")
+        descending_file = os.path.join(CURRENT_DIR, "descending.db")
+        print(descending_file)
```

Comment:
@robintw - we can probably delete that debug line.

Reply:
Fixed now - thanks.